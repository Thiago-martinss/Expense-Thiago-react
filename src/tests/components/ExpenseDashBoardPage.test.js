import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';


test('Deve renderizar ExpenseDashBoardPage corretamente', () => {
    const wrapper = shallow(<ExpenseDashboardPage/>);
    expect(wrapper).toMatchSnapshot();
  });
  