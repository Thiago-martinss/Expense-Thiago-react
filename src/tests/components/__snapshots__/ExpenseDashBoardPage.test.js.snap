// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar ExpenseDashBoardPage corretamente 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
